Birdui/Birdui/PostListView.swift

```diff
@@ -9,10 +9,56 @@
 import SwiftUI
 
 struct PostListView: View {
+    @State var modalIsPresented = false
+
+    struct HeaderTitleTextStyle: ViewModifier {
+        func body(content: Content) -> some View {
+            return content
+                .foregroundColor(Color.black)
+                .font(.system(size: 17))
```

Comment:
Can you please make font `.system(.title)`? This way, it'll look closer to what Audrey showed in the HW file.

Reply:
Yes no problem. In the homework of week 5, I took the same size of the storyboard texts(17 for title and 15 for create button) but I can change if you prefer.